6_temp_coop_fetch/log/6_temp_coop_fetch_tasks.ind

```diff
@@ -97,15 +98,15 @@
 6_temp_coop_fetch/in/lower_red_surface_DO_temp_2017.xlsx.ind: 97693663909ff673c35b58b7ea88720c
 6_temp_coop_fetch/in/lower_red_bottom_temp_DO_9metersdeep_2017.xlsx.ind: 94a1febd4d564b42f0c67b7eb3346a3d
 6_temp_coop_fetch/in/hubbard_mnlakedata_historicalfiles_manualentry.xlsx.ind: 0039fc7f98b34e06d60c036fde8d6a84
-6_temp_coop_fetch/in/Winnie_temp_loggers_2013.xlsx.ind: 76392c97ee043eba66f7522e020875b5
-6_temp_coop_fetch/in/Winnie_water_temp_2012.xlsx.ind: 9116b7df6d69d55519e843e9f5336a18
 6_temp_coop_fetch/in/Winnie_Temp_2010.xlsx.ind: 5167ff0a9232e5e829dfa0f16e537e85
 6_temp_coop_fetch/in/Winnie_Temp_2011.xlsx.ind: ea2ddeb745b1735d29ff8045a4d2b86c
+6_temp_coop_fetch/in/Winnie_temp_loggers_2013.xlsx.ind: 76392c97ee043eba66f7522e020875b5
 6_temp_coop_fetch/in/Winnie_temp_2014.xlsx.ind: 2074717df365d6c1253c813230aac402
-6_temp_coop_fetch/in/DNRdatarequest_Secchi_DO_and_Temp_1083_2016_AllLakes_JL.xlsx.ind: 22c8cc6fc3af2311c2d90edf9940fedb
 6_temp_coop_fetch/in/Winnie_temp_logger_output_2015.xlsx.ind: 2af920209709ea42afb3ace38e0f0bdc
 6_temp_coop_fetch/in/Winnie_temp_logger_2017.xlsx.ind: 3c93b9db2f6fae33db885a32f38302f5
+6_temp_coop_fetch/in/Winnie_water_temp_2012.xlsx.ind: 9116b7df6d69d55519e843e9f5336a18
 6_temp_coop_fetch/in/Winnie_temp_2_2009.xlsx.ind: c19ce966b9c5a6072c0cf1d1516dd2f2
+6_temp_coop_fetch/in/DNRdatarequest_Secchi_DO_and_Temp_1083_2016_AllLakes_JL.xlsx.ind: 22c8cc6fc3af2311c2d90edf9940fedb
```

Comment:
I created [this scipiper issue](https://github.com/USGS-R/scipiper/issues/153) and I think the ordering that matters here is in `sc_indicate` instead of the result from `drive_ls`, since this `.ind` file is the result of the task table and not of the GD inventory function. 

But yes, you are seeing the re-ordering, not a new hash. It is distracting to see diffs when the hashes aren't changing. I'm with ya